### PR TITLE
Small clean ups on forums controller

### PR DESCRIPTION
**Pull request by Br3nda**

### CODE REVIEW

**Comment by Br3nda** [comment]
specs are passing, but codeclimate is finding more duplicate code each time i change something.

**Comment by Br3nda** [comment]
Codeclimate is claiming this causes duplicate code between posts and plant parts.
neither files are changed in here -- so nope. Codeclimate you're dreaming.

**Comment by pozorvlak** [comment]
I think what's happening there is that those duplications have been removed in the `dev` branch, and CodeClimate is comparing this to `dev` rather than to the most recent common ancestor. So if I update the branch with the latest `dev` changes, CodeClimate should be happy again. And in fact it's just gone green as I was typing this :-)

File: app/controllers/forums_controller.rb [hunk 1]
```diff
@@ -1,86 +1,51 @@
...
-      format.json { render json: @forum }
-    end
+    respond_with(@forum)
   end
 
   # GET /forums/1/edit
-  def edit
-    @forum = Forum.find(params[:id])
-  end
+  def edit; end
```
Comment (pozorvlak) [inline app/controllers/forums_controller.rb]: Do we need to define this at all?

**Pull request merged**